### PR TITLE
glossary: new update

### PR DESCRIPTION
**Pull request by Blind4Basics**

### CODE REVIEW

File: data/glossary.yml [hunk 1]
```diff
@@ -238,8 +241,8 @@
     A retired Kata is a Kata that has been judge not qualitative enough by other codewarriors during the beta process. It has been removed from the beta process after some critical thresholds have been reached, without any action of the author. Nothing can be done anymore with a retired Kata.
 
 - term: Sample tests
-  description: |
-    See example tests.
+  description: ""
```
Comment (Blind4Basics) [inline data/glossary.yml:244]:
```suggestion
  description: "meh?"
```

File: data/glossary.yml [hunk 2]
```diff
@@ -146,14 +162,15 @@
     Note that the `Issue` label is ___NOT___ to use for a user who doesn't understand why his solution is failing some (or all) the tests of the kata.
 
 - term: Kata
-  description: 型
+  description: "Kata" (型) is how coding tasks are named on Codewars. See [here - TODO]() for more information about it.
```
Comment (Blind4Basics) [inline data/glossary.yml:165]:
```suggestion
  description: |
    "Kata" (型) is how coding tasks are named on Codewars. See [here - TODO]() for more information about it.
```

File: data/glossary.yml [hunk 3]
```diff
@@ -73,13 +75,16 @@
...
 
 - term: Contributor(s)
   description: User(s) who have either authored, approved, translated or modified a kata in any way (a list of them can be seen in the "Details" page of any kata.
 
 - term: Dan
-  description: 段
+  description: |
+    (段) - The term is borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go.
+    Dan indicates the number of degrees in master level. This is why they count upward.
+  see: /gamification/ranks/
```
Comment (Blind4Basics) [inline data/glossary.yml:87]:
```suggestion
  see: /concepts/gamification/ranks/
```

File: data/glossary.yml [hunk 4]
```diff
@@ -186,6 +207,7 @@
 - term: Markdowns (formatting)
   description: |
     Codewars is responsive to `markdowns`. They allow you to format code, descriptions or comments so that thay are properly readable for other users.
+  see: /markdown/
```
Comment (Blind4Basics) [inline data/glossary.yml:210]:
```suggestion
 see: /references/markdown/
```

File: data/glossary.yml [hunk 5]
```diff
@@ -207,6 +235,7 @@
 - term: Progress
   description: |
     Aside of honor points, you gain "progress" by completing Kata (and only that way). Progess is what is conditioning your current rank as a codewarrior.
+  see: /gamification/ranks/
```
Comment (Blind4Basics) [inline data/glossary.yml:238]:
```suggestion
 see: /concepts/gamification/ranks/
```

File: data/glossary.yml [hunk 6]
```diff
@@ -269,13 +303,18 @@
     - title: Test-driven Development - Wikipedia
       url: https://en.wikipedia.org/wiki/Test-driven_development
 
+- term: Test suite (kata)
+  description: See [test cases](/glossary/#test-cases-id)
+
 - term: Trainer (kata)
   description: |
     This is the place where you write your code to complete a task.
+  see: /solving-kata/
```
Comment (Blind4Basics) [inline data/glossary.yml:312]:
```suggestion
 see: /getting-started/solving-kata/
```

File: data/glossary.yml [hunk 7]
```diff
@@ -269,13 +303,18 @@
...
 
 - term: Trainer suggestions
   description: |
     Trainer suggestions are the Kata that are showing up at the top of the dashboard. Those suggestions can be tweaked at some point, either through the profile of the user or directly through the dashboard.
+    See [training setup](/setting-up/#training-setup).
```
Comment (Blind4Basics) [inline data/glossary.yml:317]:
```suggestion
    See [training setup](/getting-started/setting-up/#training-setup).
```

File: data/glossary.yml [hunk 8]
```diff
@@ -186,6 +206,7 @@
 - term: Markdowns (formatting)
   description: |
     Codewars is responsive to `markdowns`. They allow you to format code, descriptions or comments so that thay are properly readable for other users.
+ see: /references/markdown/
```
Comment (Blind4Basics) [inline data/glossary.yml:209]:
```suggestion
    see: /references/markdown/
```

**Comment by Blind4Basics** [comment]
OK, ready for review.

I tried some things along the way (aside from those see/link stuffs), and it looks like the css would benefit from some changes:

* currently:
  - when using a new paragraph in a definition, the space between the two lines/paragraphs is rather 'big"
  - this spacing is exactly the same between 2 § of the same term than the one btetween the title of a term and the text

* suggestion:
  - reduce the space betwee § (should just be a line feed, actually, no additional space)
  - enforce "several line feeds" inbetween different terms 

here is the issue with the list of new terms added/needing review (a lot of old ones still need a check): #23

**Comment by kazk** [comment]
Spacing between paragraphs are from the plugin I trust and wouldn't want to change it. I'll increase the spacing between terms.

**Comment by Blind4Basics** [comment]
should we approve this one or you need to stalk through the definitions yet?

File: data/glossary.yml [hunk 9]
```diff
@@ -73,13 +74,17 @@
...
 
 - term: Contributor(s)
   description: User(s) who have either authored, approved, translated or modified a kata in any way (a list of them can be seen in the "Details" page of any kata.
 
 - term: Dan
-  description: 段
+  description: |
+    (段) - The term is borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go.
+
+    Dan indicates the number of degrees in master level. This is why they count upward.
```
Comment (kazk) [inline data/glossary.yml:86]:
```suggestion
    Dan (段) indicates the number of degrees in master level. This is why they count upward.

    The term is borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go.
```

File: data/glossary.yml [hunk 10]
```diff
@@ -162,7 +188,11 @@
       url: https://en.wikipedia.org/wiki/Kumite
 
 - term: Kyū
-  description: 級
+  description: |
+    (級) - The term is borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go.
+
+    Kyu (or Kyū) indicates the number of degrees away from master level (Dan). This is why they count downward.
```
Comment (kazk) [inline data/glossary.yml:194]:
```suggestion
    Kyū (級) indicates the number of degrees away from master level (Dan). This is why they count downward.

    The term is borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go.
```

File: data/glossary.yml [hunk 11]
```diff
@@ -143,17 +165,21 @@
 - term: Issue (comment)
   description: |
     `Issue` is the label to use when you write a comment in the discourses section of a kata to point out some problems in the kata itself (wrong tests, wrong reference solution, wrong description, missing tests, misleading description, ...).
+
     Note that the `Issue` label is ___NOT___ to use for a user who doesn't understand why his solution is failing some (or all) the tests of the kata.
 
 - term: Kata
-  description: 型
+  description: |
+    "Kata" (型) is how coding tasks are named on Codewars. See [here - TODO]() for more information about it.
```
Comment (kazk) [inline data/glossary.yml:173]:
```suggestion
    Kata (型) is how coding tasks are named on Codewars. See [here - TODO]() for more information about it.
```

File: data/glossary.yml [hunk 12]
```diff
@@ -92,18 +97,30 @@
 
 - term: Draft (kata)
   description: |
-    Initial state of a new born kata: when you create a new kata, it is in Draft mode as long as it is not "published". Kata in draft state are not visible through the [katas search page](https://www.codewars.com/kata/my-languages), but can be accessed and tried by other users if the author gives them the direct link to the trainer. Note that a kata in draft state cannot be completed (eg. "submit final" will not show up if you pass all the tests of a draft).
+    Initial state of a new born kata: when you create a new kata, it is in Draft mode as long as it is not "published". 
+
+    Kata in draft state are not visible through the [katas search page](https://www.codewars.com/kata/my-languages), but can be accessed and tried by other users if the author gives them the direct link to the trainer. Note that a kata in draft state cannot be completed (eg. "submit final" will not show up if you pass all the tests of a draft).
```
Comment (kazk) [inline data/glossary.yml:102]:
```suggestion
    Kata in draft state are not visible through the [kata search page](https://www.codewars.com/kata/my-languages), but can be accessed and tried by other users if the author gives them the direct link to the trainer. Note that a kata in draft state cannot be completed (eg. "submit final" will not show up if you pass all the tests of a draft).
```

**Pull request merged**